framework/src/org/checkerframework/common/value/ValueAnnotatedTypeFactory.java

```diff
@@ -1151,7 +1229,7 @@ public AnnotationMirror createArrayLenAnnotation(List<Integer> values) {
         if (values.isEmpty() || Collections.min(values) < 0) {
```

Comment:
Minor: seems to be a small negligence in the original code, but the `values.isEmpty()` check seems to be unnecessary, does it?